app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -327,7 +360,8 @@ public void noInternet(NoInternetEvent event) {
     @Subscribe
     public void cantDownload(CantDownloadEvent event) {
```

Comment:
Why are there 2 events: `CantDownloadEvent` & `ErrorDialogEvent` I don't there should be 2 events for 1 point of failure, unless you have 2 different UI components or some background processing, you should consider using only one and remove the other.